### PR TITLE
Update Commerce App to Material 3

### PR DESCRIPTION
**Pull request by carldebilly**

### CODE REVIEW

File: samples/Commerce/Commerce.UI/Styles/FeedView.xaml [hunk 1]
```diff
@@ -37,7 +37,7 @@
 				<DataTemplate>
 					<not_skia:Grid>
 						<winui:ProgressRing
-							Style="{StaticResource MaterialProgressRingStyle}"
+							Style="{StaticResource ProgressRingStyle}"
```
Comment (kazo0) [inline samples/Commerce/Commerce.UI/Styles/FeedView.xaml:40]: This is good for now but we should make a pass afterwards to remove the Style setters for the styles that are implicit by default

File: samples/Commerce/Commerce.UI/App.xaml [hunk 2]
```diff
@@ -23,18 +23,18 @@
 				<ToolkitResources xmlns="using:Uno.Toolkit.UI" />
 
 				<!-- Load Material Toolkit resources -->
-				<MaterialToolkitResourcesV1 xmlns="using:Uno.Toolkit.UI.Material" />
+				<MaterialToolkitResources xmlns="using:Uno.Toolkit.UI.Material" />
```
Comment (kazo0) [inline samples/Commerce/Commerce.UI/App.xaml:26]: You'll need to change the reference of `MaterialResourcesV1` to `MaterialResources` at line 22

Comment (kazo0) [inline samples/Commerce/Commerce.UI/App.xaml:26]: Also, we'll need to edit the ColorPaletteOverride.xaml to conform to the new color keys

File: samples/Commerce/Commerce.UI/Views/CartPage.xaml [hunk 3]
```diff
@@ -195,7 +195,7 @@
 												x:Uid="_Checkout"
 												uen:Navigation.Request="Checkout"
 												utu:AutoLayout.CounterAlignment="Stretch"
-												Style="{StaticResource MaterialContainedButtonStyle}" />
```
Comment (kazo0) [inline samples/Commerce/Commerce.UI/Views/CartPage.xaml:198]: There is no Contained button style anymore, would be Filled or Elevated

File: samples/Commerce/Commerce.UI/Views/FilterPage.xaml [hunk 4]
```diff
@@ -36,70 +36,70 @@
...
 						<utu:Chip Content="Accessories"
 								  x:Uid="Body_Accessproes"
 								  IsChecked="{Binding Filter.Accessories, Mode=TwoWay}"
-								  Style="{StaticResource MaterialOutlinedFilterChipStyle}" />
+								  Style="{StaticResource OutlinedFilterChipStyle}" />
 						<utu:Chip Content="Headwear"
 								  x:Uid="Body_Headwear"
 								  IsChecked="{Binding Filter.Headwear, Mode=TwoWay}"
-								  Style="{StaticResource MaterialOutlinedFilterChipStyle}" />
+								  Style="{StaticResource OutlinedFilterChipStyle}" />
```
Comment (kazo0) [inline samples/Commerce/Commerce.UI/Views/FilterPage.xaml:56]: No more Outlined, it's either FilterChipStyle or ElevatedFilterChipStyle

https://m3.material.io/components/chips/guidelines#8d453d50-8d8e-43aa-9ae3-87ed134d2e64

File: samples/Commerce/Commerce.UI/Views/FilterPage.xaml [hunk 5]
```diff
@@ -110,7 +110,7 @@
 				<Button Content="Apply"
 						x:Uid="Body_Apply"
 						utu:AutoLayout.PrimaryAlignment="Stretch"
-						Style="{StaticResource MaterialContainedButtonStyle}"
+						Style="{StaticResource ContainedButtonStyle}"
```
Comment (kazo0) [inline samples/Commerce/Commerce.UI/Views/FilterPage.xaml:113]: Contained is now either Elevated, Filled, or Filled Tonal

https://m3.material.io/components/buttons/overview

File: samples/Commerce/Commerce.UI/Views/HomePage.xaml [hunk 6]
```diff
@@ -153,7 +153,7 @@
 									AutomationProperties.Name="Cart"
 									utu:AutoLayout.PrimaryAlignment="Stretch"
 									utu:AutoLayout.CounterLength="56"
-									Style="{StaticResource MaterialBottomTabBarItemFabStyle}"
+									Style="{StaticResource BottomTabBarItemFabStyle}"
```
Comment (kazo0) [inline samples/Commerce/Commerce.UI/Views/HomePage.xaml:156]:
```suggestion
									Style="{StaticResource BottomFabTabBarItemStyle}"
```

File: samples/Commerce/Commerce.UI/Views/LoginPage.xaml [hunk 7]
```diff
@@ -9,60 +9,60 @@
...
 								 x:Uid="Body_HyperlinkButton"
 								 uen:Navigation.Request="!Forgot"
-								 Style="{StaticResource MaterialHyperlinkButtonStyle}" />
+								 Style="{StaticResource HyperlinkButtonStyle}" />
 				<Button Content="Login"
 						AutomationProperties.AutomationId="Login"
 						x:Uid="Body_Login"
 						utu:AutoLayout.CounterAlignment="Stretch"
-						Style="{StaticResource MaterialContainedButtonStyle}"
+						Style="{StaticResource ContainedButtonStyle}"
```
Comment (kazo0) [inline samples/Commerce/Commerce.UI/Views/LoginPage.xaml:51]: Contained is now either Elevated, Filled, or Filled Tonal

https://m3.material.io/components/buttons/overview

File: samples/Commerce/Commerce.UI/Views/ProductControl.xaml [hunk 8]
```diff
@@ -90,32 +90,32 @@
 								<Button Content="add to cart"
 										x:Uid="_AddToCart"
 										utu:AutoLayout.CounterAlignment="Stretch"
-										Style="{StaticResource MaterialContainedButtonStyle}" />
+										Style="{StaticResource ContainedButtonStyle}" />
```
Comment (kazo0) [inline samples/Commerce/Commerce.UI/Views/ProductControl.xaml:93]: Contained is now either Elevated, Filled, or Filled Tonal

https://m3.material.io/components/buttons/overview

File: samples/Commerce/Commerce.UI/Views/ProductControl.xaml [hunk 9]
```diff
@@ -174,7 +174,7 @@
 																Orientation="Horizontal"
 																utu:AutoLayout.CounterAlignment="Stretch">
 													<ToggleButton utu:AutoLayout.CounterAlignment="Center"
-																  Style="{StaticResource MaterialToggleButtonIconStyle}">
+																  Style="{StaticResource ToggleButtonIconStyle}">
```
Comment (kazo0) [inline samples/Commerce/Commerce.UI/Views/ProductControl.xaml:177]:
```suggestion
																  Style="{StaticResource IconToggleButtonStyle}">
```

File: samples/Commerce/Commerce.UI/Views/ProductsPage.xaml [hunk 10]
```diff
@@ -90,21 +90,21 @@
...
 										 x:Uid="Page_Filters"
 										 utu:AutoLayout.CounterAlignment="Center"
-										 Style="{StaticResource MaterialHyperlinkButtonStyle}"
+										 Style="{StaticResource HyperlinkButtonStyle}"
 										 uen:Navigation.Request="!Filter"
 										 uen:Navigation.Data="{Binding Filter.Value, Mode=TwoWay}" />
 					</utu:AutoLayout>
 				</utu:AutoLayout>
-				<uer:FeedView Style="{StaticResource MaterialCompositeFeedViewStyle}"
+				<uer:FeedView Style="{StaticResource CompositeFeedViewStyle}"
```
Comment (kazo0) [inline samples/Commerce/Commerce.UI/Views/ProductsPage.xaml:107]: This is a custom style defined in the app, and I believe it still has a Material prefix at this point

File: samples/Commerce/Commerce.UI/Styles/FeedView.xaml (hunk 1 above)
Comment (carldebilly) [reply samples/Commerce/Commerce.UI/Styles/FeedView.xaml:40]: My goal is to do the migration first and check for those details afterwards.

File: samples/Commerce/Commerce.UI/App.xaml (hunk 2 above)
Comment (carldebilly) [reply samples/Commerce/Commerce.UI/App.xaml:26]: Yes, but for that we'll take the new colors from the Figma document, which is not ready yet.

**Pull request merged**